acceptance/lib/helper.rb

```diff
@@ -160,7 +160,7 @@ def install_puppet_server (host, make_env={})
   # setting should be able to be moved into a regular conf file.  See
   # SERVER-331.
   def get_rubylibdir host, config_key
-    "/opt/puppetlabs/agent/lib/ruby/vendor_ruby"
+    "/opt/puppetlabs/puppet/lib/ruby/vendor_ruby"
   end
 
   def configure_puppet_server
```

Comment:
Are the `/etc/puppetserver` references in here still accurate or should they be `/etc/puppetlabs/puppetserver` instead?